regtech_mail_api/api.py

```diff
@@ -77,10 +77,10 @@ async def get_debug_info(request: Request):
 async def send_email(request: Request):
     headers = request.headers
     subject = headers["X-Mail-Subject"]
-    sender_addr = headers["X-Mail-Sender-Address"]
-    sender_name = headers["X-Mail-Sender-Name"]
+    sender_addr = request.user.email
+    sender_name = request.user.name
 
-    sender = f"{sender_name} <{sender_addr}>" if sender_addr else sender_name
+    sender = (f"{sender_name} <{sender_addr}>" if sender_addr else sender_name).strip()
```

Comment:
Did I screw up the logic here? Shouldn't this be...
```suggestion
    sender = (f"{sender_name} <{sender_addr}>" if sender_name else sender_addr).strip()
```

I'm guessing that'd fix the `<>` in the `sender` value question I asked too?

...and if we do that, do we still need the `strip()`. I'm not sure what problem that was fixing.

Reply:
The strip is there because if the name is empty you end up with an empty space prior to the <>.   Name ends up either be filled or an empty string (it will never be null) so I was stripping off extra white space.  Now if name is empty and we instead just want the email address sans <> then I can make that just be the plain email.

Reply:
Ah yeah didn’t even realize that, can take the strip off if (and it would lose the <>) it’s set to send_addr when send_name is empty. Dur. Will fix.